java/TravelSample/src/main/java/com/couchbase/travelsample/model/Flight.java

```diff
@@ -46,7 +48,7 @@ public static Flight fromJSON(@Nullable JSONObject json) {
                 (!json.has(PROP_EQUIPMENT)) ? null : json.getString(PROP_EQUIPMENT),
                 (!json.has(PROP_DEPARTURE_TIME)) ? null : json.getString(PROP_DEPARTURE_TIME),
                 (!json.has(PROP_FLIGHT_TIME)) ? 0 : json.getInt(PROP_FLIGHT_TIME),
-                (!json.has(PROP_PRICE)) ? 0F : json.getFloat(PROP_PRICE));
+                (!json.has(PROP_PRICE)) ? 0F : json.getLong(PROP_PRICE));//there's no getFloat
```

Comment:
Yes there is!
https://stleary.github.io/JSON-java/org/json/JSONObject.html#getFloat-java.lang.String-

There's something odd going on, here...